tool/site/client/src/pages/dispensePage.tsx

```diff
@@ -162,6 +162,8 @@ export function createStaticLineItemInfo(
     lineItemInfo.priorStatusCode = getLineItemStatus(latestMedicationDispense)
   }
 
+  lineItemInfo.alternativeMedicationAvailable = containsParacetamol(medicationRequest)
+
```

Comment:
on this page when the dispense result shows, if the prescription is still with dispenser - active we should have a `dispense` prescription action available